sdk/src/vite/useClientPlugin.mts

```diff
@@ -288,12 +290,19 @@ export const useClientPlugin = (): Plugin => ({
       return;
     }
 
-    if (code.includes('"use client"') || code.includes("'use client'")) {
-      const relativeId = `/${relative(this.environment.getTopLevelConfig().root, id)}`;
+    const cleanCode = code.trimStart();
+    if (
+      cleanCode.startsWith('"use client"') ||
+      cleanCode.startsWith("'use client'")
+    ) {
```

Comment:
This is all that I changed, I'm not sure why my prettier rules differ; let me see if I can fix this.

Reply:
Oh right! We just need to check the start!